Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tikv"
-version = "1.1.0-dev"
+version = "1.1.0-dev.1"
```

Comment:
Please update it in an independent PR

Reply:
Because this PR needs to bind a new version that can make TiDB knows it.